src/components/menu/MenuHeader.scss

```diff
@@ -20,6 +20,8 @@
 }
 
 .bdl-MenuHeader-title {
+    @include bdl-lineClamp;
```

Comment:
is there a conflict with using `line-clamp` and `text-overflow: ellipsis` together? does it just add ellipsis to the last line of the line clamp?

Reply:
There's no conflict between the two but `text-overflow` doesn't have any effect when being used together with `bdl-lineClamp` either regardless of the value set. So I removed `text-overflow: ellipsis` as well as `overflow: hidden`, which is already assigned in `bdl-lineClamp`.

`bdl-lineClamp` automatically adds an ellipsis to the last line of text.